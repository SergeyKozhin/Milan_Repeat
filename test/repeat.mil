BEGIN
    n := READ;
    i := 0;
    REPEAT
        i := i + 1;
        WRITE(i)
    UNTIL i >= n
END
