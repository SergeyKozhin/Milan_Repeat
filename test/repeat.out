0:	INPUT
1:	STORE	0
2:	PUSH	0
3:	STORE	1
4:	LOAD	1
5:	PUSH	1
6:	ADD
7:	STORE	1
8:	LOAD	1
9:	PRINT
10:	LOAD	1
11:	LOAD	0
12:	COMPARE	5
13:	JUMP_NO	4
14:	STOP
